gammapy/modeling/serialize.py

```diff
@@ -3,11 +3,35 @@
 from gammapy.cube.fit import MapDataset
 from gammapy.spectrum import FluxPointsDataset, SpectrumDataset
 from .models import Registry, SkyDiffuseCube, SkyModel
+from regions import write_ds9
 
 # TODO: move this elsewhere ?
 DATASETS = Registry([MapDataset, SpectrumDataset, FluxPointsDataset])
 
-__all__ = ["models_to_dict", "dict_to_models", "dict_to_datasets", "datasets_to_dict"]
+__all__ = [
+    "regions_to_reg",
+    "models_to_dict",
+    "dict_to_models",
+    "dict_to_datasets",
+    "datasets_to_dict",
+]
+
+
+def regions_to_reg(regions, filename, **kwargs):
```

Comment:
Please remove this helper function.
Ideally there shouldn't be `None` sometimes, it's better to fill `NaN` and always return the same object.

Even if you keep like this, then filtering out the `None` is just one line:
```
regions = [_ for _ in regions if _ is not None]
```
Let's not introduce a function for that, just put that 1 line in the 1-2 places where you need it.